e2e/src/api/specs/person.e2e-spec.ts

```diff
@@ -141,6 +145,41 @@ describe('/activity', () => {
     });
   });
 
+  describe('POST /person', () => {
+    it('should require authentication', async () => {
+      const { status, body } = await request(app).post(`/person`);
+      expect(status).toBe(401);
+      expect(body).toEqual(errorDto.unauthorized);
+    });
+
+    it('should not accept invalid birth dates', async () => {
+      for (const { birthDate, response } of invalidBirthday) {
```

Comment:
Maybe we should move the for each out of the `it` for a more helpful output if the test fails?

Reply:
I will do it in a follow up. There is only one test that runs either way right now.